.gitignore

```diff
@@ -16,3 +16,4 @@ packages
 .directory
 Thumbs.db
 *.userprefs
+*.ffs_db
```

Comment:
The general way to change the last commit is via `git commit --amend`, but you'll need to rebase those 3 merge commits away first.